src/content/samples/node.mdx

```diff
@@ -154,7 +121,7 @@ Starting hello-world server...
 
 Go back to the browser and reload the page. Your code changes were instantly applied. No commit, build, or push required 😎!
 
-## Step 5: Debug directly on Okteto Cloud
+## Step 4: Debug directly on Okteto Cloud
```

Comment:
You are missing this change:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16L225